### ISSUE TITLE
different glibc versions

### ISSUE THREAD

**Issue by markuman** [dex4er/fakechroot#18]
I'm running Debian wheezy and want to run an Arch Linux pacstrap.

```
$ fakechroot fakeroot chroot arch/
/bin/bash: relocation error: /home/markuman/arch/usr/lib/libc.so.6: symbol _dl_find_dso_for_object, version GLIBC_PRIVATE not defined in file ld-linux-x86-64.so.2 with link time reference
```

And the other way around, I'm running Arch Linux and want to run a Debian Wheezy debootstrap

```
$ fakechroot fakeroot chroot root/
/usr/bin/chroot: /home/markus/wheezy/root/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.14' not found (required by /usr/bin/chroot)
/usr/bin/chroot: /home/markus/wheezy/root/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.14' not found (required by /usr/lib/libfakeroot/libfakeroot.so)
/usr/bin/chroot: /home/markus/wheezy/root/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.14' not found (required by /usr/lib/libfakeroot/fakechroot/libfakechroot.so)
```

Any ideas how to handle those different glibc versions?

**Comment by abliss** [comment]
In the "Limitations" section of the manpage, it says:
https://github.com/dex4er/fakechroot/blob/master/man/fakechroot.pod#limitations

- /lib/ld-linux.so.2 and /lib64/ld-linux-x86-64.so.2 are always loaded from real environment. This path is hardcoded by linker for all binaries. You can set the FAKECHROOT_ELFLOADER environment variable or use --elfloader option.
- Every command executed within fakechroot needs to be linked to the same version of the dynamic linker from real environment. If the libraries in chroot are not compatible, try to use --use-system-libs option.

So, in short, I think you're out of luck.
